ipatests/test_integration/test_ntp_options.py

```diff
@@ -1,7 +1,7 @@
 #
 # Copyright (C) 2018  FreeIPA Contributors see COPYING for license
 #
-
+from __future__ import absolute_import
```

Comment:
Note, this line is no longer necessary on master. You can leave it in if that is easier for a backport. Let's remove the absolute imports from all files together.

Reply:
Ok i will remove it and leave it in PR for 4-7 branch